docs/BoundedInts.md

```diff
@@ -0,0 +1,51 @@
+
+## The `BoundedInts` module {#sec-boundedints}
+
+The `Dafny.BoundedInts` module contains definitions of types and constants for fixed-bit-width integers.
```

Comment:
When dealing with bounded values
we also need subset types for dealing with `seq`.
This is because the minute that you touch a `seq`
or any of the other collections
you fall off the bounded value train.
So having a set of types for these things
is very powerful.

```dafny
predicate Int16?(i: int)
{
  i < T.TWO_TO_THE_15
}

type seq16<T> = s:seq<T> | Int16?(|s|)
```

Reply:
I agree, but not yet in the scope of documentation!